moveit_ros/moveit_servo/test/basic_servo_tests.cpp

```diff
@@ -170,6 +174,71 @@ TEST_F(ServoFixture, SendJointServoTest)
   EXPECT_LT(received_count, num_commands + 20);
   servo_->setPaused(true);
 }
+
+// This a friend test of a private member function
+TEST_F(ServoFixture, EnforceVelLimitsTest)
+{
+  auto parameters = servo_->getParameters();
+  const double publish_period = parameters.publish_period;
+
+  // Request joint angle changes that are too fast, given the control period in servo settings YAML file.
+  Eigen::ArrayXd delta_theta(7);
```

Comment:
i see you made this a 7-vector because the joint group has 7 joints. Good catch.